FieldOpt/Hdf5SummaryReader/hdf5_summary_reader.cpp

```diff
@@ -1,3 +1,27 @@
+/******************************************************************************
```

Comment:
This is the wrong license header. I believe we agreed that only the WIC was to be under the LGPL?